change/@fluentui-react-accordion-cc2dd8b7-5f02-4e63-b3a4-30693e0a8fb6.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30389&commitId=622ebabb2dd1550db1a18d6a9a1ca9d25f605c1c&env=prod&clientType=FLUENTUI)


  <details><summary><b>Accordion Converged</b> 16 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Accordion Converged.visibility+focus - High Contrast.focus closed.chromium.png|353| Changed|
|Accordion Converged.visibility+focus - RTL.focus closed.chromium.png|306| Changed|
|Accordion Converged.visibility+focus - RTL.focus opened.chromium.png|306| Changed|
|Accordion Converged.visibility+focus - High Contrast.focus opened.chromium.png|353| Changed|
|Accordion Converged.visibility+focus.normal.chromium.png|303| Changed|
|Accordion Converged.expandIconPosition  end .normal.chromium.png|292| Changed|
|Accordion Converged.visibility+focus - RTL.normal.chromium.png|306| Changed|
|Accordion Converged.visibility+focus - Dark Mode.normal.chromium.png|295| Changed|
|Accordion Converged.visibility+focus - High Contrast.normal.chromium.png|353| Changed|
|Accordion Converged.expandIconPosition  end  - RTL.normal.chromium.png|280| Changed|
|Accordion Converged.visibility+focus - Dark Mode.focus closed.chromium.png|295| Changed|
|Accordion Converged.visibility+focus - Dark Mode.focus opened.chromium.png|295| Changed|
|Accordion Converged.visibility+focus.focus opened.chromium.png|303| Changed|
|Accordion Converged.visibility+focus.focus closed.chromium.png|303| Changed|
|Accordion Converged.size.normal.chromium.png|1156| Changed|
|Accordion Converged.size - RTL.normal.chromium.png|1173| Changed|

</details>


  <details><summary><b>Tree</b> 14 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tree.actions.chromium.png|267| Changed|
|Tree.actions - High Contrast.chromium.png|7949| Changed|
|Tree.actions - Dark Mode.chromium.png|6608| Changed|
|Tree.actions - RTL.chromium.png|267| Changed|
|Tree.default - High Contrast.chromium.png|8139| Changed|
|Tree.default - Dark Mode.chromium.png|6736| Changed|
|Tree.default - RTL.chromium.png|409| Changed|
|Tree.expand icon - High Contrast.chromium.png|3793| Changed|
|Tree.expand icon - Dark Mode.chromium.png|3183| Changed|
|Tree.default.chromium.png|408| Changed|
|Tree.layout - High Contrast.chromium.png|8021| Changed|
|Tree.layout.chromium.png|332| Changed|
|Tree.layout - Dark Mode.chromium.png|6656| Changed|
|Tree.layout - RTL.chromium.png|332| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>